book.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'rentals'
+
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    # rental = Rental.new(date, self, person)
+    @rentals << rental
+    # person.rentals << rental
+    rental.person.rentals << rental
+  end
```

Comment:
- [ ] Kindly remove commented code.

- [ ] Also in continuation of the earlier suggestion, let's simply instantiate a rental object here and it will take care of adding itself to the `rentals` arrays.

  ```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
  ```

Reply:
#Fixed! 🔨

Reply:
done well